deploy/docker/conf/docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
     depends_on:
       - "etcd"
   etcd:
-    image: "quay.io/coreos/etcd:v3.0.13"
+    image: "quay.io/coreos/etcd:v3.0.15"
```

Comment:
Yup, I've been using v3.0.15 for over a month now. I was running into an issue with v3.0.13 with a "Context deadline is exceeded" error. Since updating to v3.0.15, I haven't seen that error repeat.